src/components/list/scss/list.scss

```diff
@@ -0,0 +1,26 @@
+$xlarge: 1200px;
```

Comment:
You should consider extrapolating all these variables you're using to a `settings.scss` file, which you then import in the other stylesheets.